aten/src/ATen/native/mps/OperationUtils.h

```diff
@@ -313,6 +313,15 @@ MPSGraphTensor* log1p(MPSGraph* mpsGraph, MPSGraphTensor* inputTensor);
      ", downcasting to a smaller data type (int32/float32). Native support for int64 has been added in macOS 13.3.");   \
   }
 
+/**
+ * Computes number of elements one needs to transfer to preserve all the elements
+ */
+size_t compute_strided_size(const at::Tensor& t);
+
+inline bool is_strided_contiguous(const at::Tensor& t) {
```

Comment:
That is a very misleading name? This only checks if the number of elements is right, not if it is actually contiguous!

Reply:
Any suggestions? (My other idea was `is_strided_dense`, but it also have a different connotations throughout the code. I can call it something like `is_non_overlapping_and_maps_to_contiguous_memory_range`, but it's a bit long, isn't it?

Reply:
tbh I prefer the super long but clear name.
Also this doesn't actually check that it is contiguous right?

I don't know what compute_strided_size computes so hard to say what a good name is!

Reply:
Hmm, how about : `compare_strided_size_and_numel`

Reply:
Renamed to `is_dense_in_storage`